test/integration_tests/long/test_model_forecasting.py

```diff
@@ -92,6 +127,7 @@ def test_forecast_with_column_rename(self):
                 WHERE bedrooms = 2
             )
             TYPE Forecasting
+            HORIZON 24
```

Comment:
This should be 12. The output is 24 because there are two values in the `type` column.